roles/email/templates/config.json.j2

```diff
@@ -1,6 +1,7 @@
 {
   "port": {{ auth_mailer_port }},
   "locales": ["en_US", "de"],
+  "defaultLanguage": ["en_US"],
```

Comment:
Someone with better understanding than me of why it's `en_US` and `de` only should answer that. @jrgm?